app/main.py

```diff
@@ -1 +1,47 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
```

Comment:
The `print_name` method is supposed to return a string according to its type hint, but it currently does not return anything. Consider returning the formatted string instead of just printing it.